scripts/redcap/purge_outdated.py

```diff
@@ -179,8 +202,15 @@ def main(api, args):
         sys.exit()
 
     marks = main(redcap_api, args)
+
+    # Log dataframe by row here, one for exceeds, and another for precedes
```

Comment:
- Instead of `description` which repeats the `message`, I'd suggest adding a `resolution` keyword instead, with the content "Contact site to determine if the date is incorrect and should be changed, if the form should be emptied, or if an exception should be set."
- `message=f"Form exceeds visit date by {args.max_days_after_visit} days"`, maybe?